ampd/src/sui/verifier.rs

```diff
@@ -22,11 +22,7 @@ impl EventType {
             EventType::ContractCall => "ContractCall",
```

Comment:
Merge this PR in coordination with other repos, so e2e tests don't break

Reply:
we don't have e2e test for this

Reply:
Joao added Sui -> Amplifier last week